### ISSUE TITLE
Training and Inference Time

### ISSUE THREAD

**Issue by II-Matto** [prstrive/UniMVSNet#3]
Thanks for sharing this excellent work!
As I have not found the description of training and inference time in the paper, I would like to ask how long it will take to train or fine-tune the model. And how long does it take to perform depth inference on 5 images (1 reference + 4 source)?
It would be so nice of you to present the training and inference time with detailed description of the settings, e.g. the type and the number of GPUs, resolution of input images, size of training set, the corresponding accuracy/completeness, etc. Many thanks.

**Comment by prstrive** [comment]
As declared in our paper, our Unification representation and Unified Focal Loss are memory-free and computation-free modules. Therefore, the computational cost and run-time of Unification+UFL is the same as our baseline CasMVSNet, and you can refer  to the Figure 1 in their paper. And if you adopt the adaptive aggregation strategy, the training time for an epoch increases by about 20 minutes from our experiments with the input size 512x640 in two Tesla V100 GPUs. The corresponding performance will also be improved to a certain extent. You can refer to our Table 3 for more detail.

**Comment by II-Matto** [comment]
Thanks for your reply. One more question: do the new task formulation and the loss have influences on the convergence speed of training? Currently the number of training epochs seems to remain the same as CasMVSNet.

**Comment by prstrive** [comment]
In fact, training for 12 epochs can already get the similar performance, so we think the convergence speed will at least not be slower than our baseline.

**Comment by BobrG** [comment]
@prstrive I wonder if you can clarify -- did you used Tesla GPUs only for training on DTU? But in order I want to finetune model on BlendedMVS according to the description in your paper -- "...we fine- tune our model on BlendedMVS for 10 epochs. We take 7 images as the input with the original size of 768 × 576." -- then two Teslas won't be enough, isn't it? 

Also, fyi you provide dataset_low_res.zip for BlendedMVS but it lacks certain directories from training_list, i.e. dataset_low_res/58c4bb4f4a69c55606122be4: No such file or directory. However I believe this dataset is similar to BlendedMVS provide in off.repository.
